### PR TITLE
Refactor/siginput

### PR DESCRIPTION
**Pull request by Delfil**
Some unit size & duplicate fixes for sig input

### CODE REVIEW

**Comment by kwendel** [comment]
looks good!

**Comment by Delfil** [comment]
(Rebase, same code)

File: README-SIG.txt [hunk 1]
```diff
@@ -13,7 +15,7 @@ Please do not grade this newick package with, because this is not our code, and
 We scored 0.6 on the component balance property at the last SIG grading. Since our project consists of loading multiple data strains and combine them to make a single graph, we need big data structures. That's why our data package is that big, it contains all the classes that hold data and classes that wrap nodes to achieve semantic zooming. After this low grade we reconsidered our package structure and came to the conclusion that moving classes and packages to balance the components doesn't make any sense, except for a ui package. It's more logical to put all of the data classes in the data package with the result that this package may be bigger than others.
 
 Contact:
-Tom Brouws 		T.H.Brouws@tudelft.nl
+Tom Brouws 			T.H.Brouws@tudelft.nl
```
Comment (mathieupost) [inline README-SIG.txt]: Why extra tab why

Comment (TomBrouws) [inline README-SIG.txt]: Yeah why

**Comment by TomBrouws** [comment]
Looks good!

File: README-SIG.txt (hunk 1 above)
Comment (Delfil) [inline README-SIG.txt]: Because it was displayed wrong in notepad++

Comment (mathieupost) [inline README-SIG.txt]: Y u use notepad++

**Pull request merged**